src/app/rpmostree-builtin-kargs.cxx

```diff
@@ -291,33 +283,53 @@ rpmostree_builtin_kargs (int            argc,
         return FALSE;
 
       const char* current_kernel_arg_string = NULL;
+      gboolean kargs_changed = FALSE;
       if (!kernel_arg_handle_editor (old_kernel_arg_string, &current_kernel_arg_string,
-                                     cancellable, error))
+                                     &kargs_changed, cancellable, error))
         return FALSE;
-      gboolean out_changed = FALSE;
+      if (!kargs_changed)
+        {
+          if (opt_unchanged_exit_77)
```

Comment:
The use case for using `--unchanged-exit-77` together with `--editor` is a bit obscure but yeah, it seems reasonable to support for consistency.

Reply:
I happened to notice because I was trying to use it to manually test the `sed` editor and I was surprised when I got wrong results.